vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionManager.java

```diff
@@ -66,4 +80,58 @@ public IExhibition getStartExhibition() {
         }
         return null;
     }
+
+    /**
+     * Updates the Exhibition with given list of languages. It fetches the language from exhibitionLanguageConfig using code.
+     *  
+     * @param exhibition
+     * @param defaultLanguage 
+     * @param languages
+     * @throws LanguageListConfigurationNotFound 
+     */
+    @Override
+    public void updateExhibitionLanguages(Exhibition exhibition, List<String> codes, String defaultLanguage) throws LanguageListConfigurationNotFound {
+        if(CollectionUtils.isEmpty(exhibitionLanguageConfig.getExhibitionLanguageList())) {
+            throw new LanguageListConfigurationNotFound("Exhibition Language Configuration not found");
```

Comment:
I think I would make `LanguageListConfigurationNotFound` a RuntimeException. If this happens it means that the installation is incomplete or something else is going wrong and there is really no good way out. So I'd make it a runtime exception and not declare or handle the exception. Then just add a handler to the `ExhibitionExceptionHandler` that handles all runtime exceptions by showing a 500 page.

Reply:
I have added it to the exception handler. The exception is thrown and not being handled. Is there any thing else to be done for this?